### PR TITLE
zed: merge yoga

### PR DESCRIPTION
**Pull request by markgoddard**
- Restore upstream mirrors in container image package repos
- Fix for ci-aio automated deploy on non-lvm images
- Update CentOS 8 Stream snapshots
- Exclude broken iptables from latest snapshot
- Reduce service workers for aufn-ceph & virtualised ci-multinode envs
- multinode: Run RefStack Tempest tests by default
- Fix growroot.yml on non-LVM images
- Minor growroot.yml fixes
- Bump container image tags
- Revert original non-LVM deployment fix
- Add bifrost image tag
- Pin all iptables packages
- Fixup for previous commit
- growroot.yml conditonal fix
- Adds Centos 8 Stream Extras Common
- Update release note
- Fixups for missing references to extras common
- Further fixups for path consistency
- Make identifier and name match existing in .repo file
- rebuild opensearch images
- Bump images to switch to extras common
- Add physical size to nvmemon metrics
- Add release note
- Change skip_lvm_check to ignore_lvm_check
- Revert container image bump to nova libvirt
- Revert growpart check
- Mention switch from extras to extras common
- fix nova-libvirt tag
- Update etc/kayobe/kolla/globals.yml
- Update kibana
- Set kolla tag in base config
- Fix repo URL
- Bump ubuntu snapshots
- Make sure kibana is built in standard set of images
- New snapshots
- Bump ubuntu images
- Add missing linux-firmware package in Rocky Linux 9
- Fix nova_libvirt pin
- CI: Only sync old images after build
- Bump image tag
- Add release notes
- Clarify release notes
- Mention the need to reboot.
- fix build of rocky images
- fix ubuntu build
- Update pulp-repo-versions.yml
- prometheus: Drop some version overrides
- CI: Set selinux_state to disabled for ci-builder environment

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -251,11 +262,30 @@ kolla_build_blocks:
...
     RUN \
-        rm -f /etc/apt/sources.list && \
+        mv /etc/apt/sources.list /etc/apt/sources.list.backup && \
     {% for repo in stackhpc_ubuntu_jammy_repos %}
         echo '{{ repo }}' >> /etc/apt/sources.list {% if not loop.last %} && \
     {% endif %}
     {% endfor %}
+  footer: |
+    {% if stackhpc_kolla_clean_up_repo_mirrors | bool %}
+    {% if kolla_base_distro == 'centos' %}
```
Comment (Alex-Welsh) [inline etc/kayobe/kolla.yml]: no more centos

File: etc/kayobe/pulp-repo-versions.yml [hunk 2]
```diff
@@ -1,30 +1,30 @@
 ---
 # Do not edit! This file is autogenerated by Ansible.
-stackhpc_pulp_repo_centos_stream_9_docker_version: 20230228T044432
-stackhpc_pulp_repo_centos_stream_9_nfv_openvswitch_version: 20230510T072502
-stackhpc_pulp_repo_centos_stream_9_openstack_zed_version: 20230721T005746
-stackhpc_pulp_repo_centos_stream_9_opstools_version: 20230301T034123
+stackhpc_pulp_repo_centos_stream_9_docker_version: 20230919T015626
+stackhpc_pulp_repo_centos_stream_9_nfv_openvswitch_version: 20230829T005959
+stackhpc_pulp_repo_centos_stream_9_openstack_zed_version: 20230919T015626
```
Comment (Alex-Welsh) [inline etc/kayobe/pulp-repo-versions.yml:5]: Not sure why this is changing in a yoga/zed merge

**Review by Alex-Welsh — CHANGES_REQUESTED** [review]

File: etc/kayobe/pulp-repo-versions.yml (hunk 2 above)
Comment (Alex-Welsh) [reply etc/kayobe/pulp-repo-versions.yml:5]: Nevermind, saw the fresh commits now

File: etc/kayobe/kolla.yml (hunk 1 above)
Comment (mnasiadka) [reply etc/kayobe/kolla.yml]: There is CentOS, we're just not using it :)

Comment (markgoddard) [reply etc/kayobe/kolla.yml]: Good spot. Fixed.

**Comment by Alex-Welsh** [comment]
CI will fail until https://github.com/stackhpc/kayobe-automation/pull/36 and https://github.com/stackhpc/stackhpc-kayobe-config/pull/629 merge

**Comment by Alex-Welsh** [comment]
This CI failure is an interesting one https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6263355888/job/17007765357?pr=654#step:21:959 I haven't been looking at it in a while because I'm expecting it to fail, but I haven't seen it go like that before

**Pull request merged**